server/leafnode_test.go

```diff
@@ -2185,65 +2185,80 @@ func TestLeafNodeLoopDetectedDueToReconnect(t *testing.T) {
 	checkLeafNodeConnected(t, sl)
 }
 
-func TestLeafNodeTwoRemotesBindToSameAccount(t *testing.T) {
+func TestLeafNodeTwoRemotesBindToSameHubAccount(t *testing.T) {
 	opts := DefaultOptions()
 	opts.LeafNode.Host = "127.0.0.1"
 	opts.LeafNode.Port = -1
 	s := RunServer(opts)
 	defer s.Shutdown()
 
-	conf := `
-	listen: 127.0.0.1:-1
-	cluster { name: ln22, listen: 127.0.0.1:-1 }
-	accounts {
-		a { users [ {user: a, password: a} ]}
-		b { users [ {user: b, password: b} ]}
-	}
-	leafnodes {
-		remotes = [
-			{
-				url: nats-leaf://127.0.0.1:%d
-				account: a
+	for _, test := range []struct {
+		name    string
+		account string
+		fail    bool
+	}{
+		{"different local accounts", "b", false},
+		{"same local accounts", "a", true},
+	} {
+		t.Run(test.name, func(t *testing.T) {
+			conf := `
+			listen: 127.0.0.1:-1
+			cluster { name: ln22, listen: 127.0.0.1:-1 }
+			accounts {
+				a { users [ {user: a, password: a} ]}
+				b { users [ {user: b, password: b} ]}
 			}
-			{
-				url: nats-leaf://127.0.0.1:%d
-				account: b
+			leafnodes {
+				remotes = [
+					{
+						url: nats-leaf://127.0.0.1:%[1]d
+						account: a
+					}
+					{
+						url: nats-leaf://127.0.0.1:%[1]d
+						account: %s
+					}
+				]
 			}
-		]
-	}
-	`
-	lconf := createConfFile(t, []byte(fmt.Sprintf(conf, opts.LeafNode.Port, opts.LeafNode.Port)))
+			`
+			lconf := createConfFile(t, []byte(fmt.Sprintf(conf, opts.LeafNode.Port, test.account)))
 
-	lopts, err := ProcessConfigFile(lconf)
-	if err != nil {
-		t.Fatalf("Error loading config file: %v", err)
-	}
-	lopts.NoLog = false
-	ln, err := NewServer(lopts)
-	if err != nil {
-		t.Fatalf("Error creating server: %v", err)
-	}
-	defer ln.Shutdown()
-	l := &captureErrorLogger{errCh: make(chan string, 10)}
-	ln.SetLogger(l, false, false)
+			lopts, err := ProcessConfigFile(lconf)
```

Comment:
Does this denote formatting spaces or something?

Reply:
This is because I transformed this test from a single to a t.Run() to cover cases where it should work (different local accounts) and should not (same local account). So the "old" body was shifted to the right.